dev-overrides.yaml

```diff
@@ -1,17 +1,13 @@
 schema_version: 1
 
-name: "jboss-eap-7-tech-preview/eap72-openshift"
-
 modules:
       repositories:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
                   ref: master
           - name: jboss-eap-modules
-            git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: master
+            path: /home/bstansberry/dev/openshift/jboss-eap-modules
```

Comment:
Probably needs resetting to the default.

Reply:
Oops.